crates/sui-cluster-test/src/faucet.rs

```diff
@@ -30,8 +30,7 @@ impl FaucetClientFactory {
                     .expect("Expect local faucet key for local cluster")
                     .copy();
                 let wallet_context = new_wallet_context_from_cluster(cluster, key)
-                    .instrument(info_span!("init_wallet_context_for_faucet"))
-                    .await;
+                    .instrument(info_span!("init_wallet_context_for_faucet")).await;
```

Comment:
i think we can drop the await here since the function is not async anymore

Reply:
yes, i think i can ues ```let wallet_context = new_wallet_context_from_cluster(cluster, key);``` directly.